Sample-01/Dockerfile

```diff
@@ -9,7 +9,7 @@ WORKDIR /app
 COPY package.json .
 COPY package-lock.json .
 
-RUN npm install
+RUN npm install --legacy-peer-deps
```

Comment:
Needed to use `--legacy-peer-deps` because our fontawesome dependency is not updated for Angular 15 yet.